sdk/internal/hashing.py

```
@@ -0,0 +1,7 @@
+import base64
+import hashlib
+
+
+def hash_unit(unit: str) -> str:
+    dig = hashlib.md5(unit.encode('utf-8')).digest()
```

Ruff (0.15.21)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)